src/api/history/history.controller.js

```diff
@@ -0,0 +1,205 @@
+const data = {
+  historyData: {
+    events: [
+      {
+        _id: 0,
+        type: "milestone",
+        data: {
+          name: "Coding Garden Created!",
+          date: "13/10/2015",
```

Comment:
Make sure there is a consistent date format. On the contributors data, we use `yyyy/mm/dd`.